components/channel_notifications_modal/components/notification_section.jsx

```diff
@@ -6,7 +6,7 @@ import React from 'react';
 
 import {NotificationSections, NotificationLevels} from 'utils/constants';
 
-import CollapseView from './collapse_view.jsx';
+import CollapseView from './collapse_view.tsx';
```

Comment:
sorry i missed this we dont need the extension here

Reply:
@M-ZubairAhmed Done